README.md

```diff
@@ -31,15 +31,15 @@ public function registerBundles()
 
 ## Commands
 
-Delete expired nonces via the ``escape:wsseauthentication:nonces:delete`` command that ships with this bundle; it takes the firewall name as a (required) parameter.
+Delete expired nonces via the `escape:wsseauthentication:nonces:delete` or alias `escape:w:n:d` command that ships with this bundle; it takes the firewall name as a (required) parameter.
```

Comment:
I'd actually leave out the aliased version of the command here... What do you think?